src/label/labelLayout.ts

```diff
@@ -361,9 +359,13 @@ export interface Label extends interactivitySelectionService.SelectableDataPoint
     backgroundColor?: string;
 
     backgroundTransparency?: number;
+
+    selected: boolean;
+
+    identity: ISelectionId;
```

Comment:
Let's create an interface as it was in interactivityutils

Reply:
Also we will be able to use this interface in `src/dataLabel/dataLabelUtils.ts`

Reply:
done e9a0ea20962684b1aab1ae52e72eb409c1f1bf57